### ISSUE TITLE
General thoughts on direction

### ISSUE THREAD

**Issue by ninjabear** [10xers/2015#3]
I don't have a specific idea yet, but I do have some ideas on the types of criteria I think a good project would fit. 
1. Something which could monopolise a niche

--Picture this, it's 1939 - winter. You're Finland. Russia is using german antics as cover for a traditional land grab. They outnumber you massively, in both soldiers + materiel. The finnish know this, but realise the weakness is in command/control and local knowledge. They devise a strategy to use economies of force - squeezing russian units into blobs. They then use indirect fire and general badassery to keep them in these blobs (which they called a mottis). Now they were isolated, they used few troops to hold most of the mottis together, freeing up enough force to defeat individual mottis in detail.

--Soo sparing the epic history lesson of the winter war, to make some headway against a slow moving monster like the corporate world, we need a problem that we can squeeze into a motti, and apply full force to monopolise it, ignoring big picture shit.  Then you can move to the next motti and so on.
1.  Under 10,000 LOC

--Even if we all pushed out 5k+ lines in a week, the result would be a bit shit. I'm not saying if we get to 10,000 stop, but we should at least estimate before we start.
1. Can't violate contracts any of us hold currently. For me and matt that probably means steering clear of aviation, though we should take a look. Sectors like that are a bit old worldly, not much "disruption" as valley would call it.
2. Something everyone can understand the point of. If people can't even appreciate in what situation it is a problem, its pointless. (like if its not broken don't bother).
3. Something we can show at the end of it. Kind of relates to 2, but I think we should get to something roughly demoable by the end of the week, since work will probably stop forever on it at that point.

**Comment by ninjabear** [comment]
github, how does it work. Whatever

**Comment by MatthewJWalls** [comment]
Probably also:
- Something you wouldn't mind actively working on if it went big. Something you would enjoy.
- Doesn't require an "in" with some field that we don't have. Example would be: urban planning software, because we'd need to work closesly with people in that field so you can't just slam down some code and hope they might find it usable after the fact.
- Not in a legal grey area unless there's a case for it being legit.

**Comment by MatthewJWalls** [comment]
I don't know if we want to do anything in terms of rules for IP, but I'm assuming basically we go by the "don't be a twat" rule.

**Comment by ninjabear** [comment]
Agreed. In fields might be a bit of an advantage if we do have someone "in" the field closeby

**Comment by ninjabear** [comment]
Closing as no longer relevant
